doozerlib/cli/release_calc_upgrade_tests.py

```diff
@@ -18,3 +19,15 @@ def release_calc_upgrade_tests(version):
     prev_versions, prev_edges = util.get_channel_versions(prev_candidate_channel, arch)
     curr_versions, current_edges = util.get_channel_versions(candidate_channel, arch)
 
+    prev_prev = f'{major}.{minor - 2}'
+    prev_versions = [x for x in prev_versions if prev_prev not in x]
+    version_list = util.sort_semver(set(prev_versions + curr_versions))
+
+    # if 'hotfix' in version:
+    #     def valid(v): return all([x not in v for x in ['rc', 'fc']])
+    #     version_list = list(filter(valid, version_list))
+    if not (('rc' in version) or ('fc' in version)):
+        def valid(v): return all([x not in v for x in ['rc', 'fc', 'hotfix']])
```

Comment:
i was considering whether `hotfix` should be eligible for testing (it's included in edges). i guess tests are not needed (can be manually requested if needed).

FWIW this leaves `hotfix` in for testing with FCs and RCs...